app/src/main/java/com/example/footballmanager/data/AppModule.kt

```diff
@@ -35,4 +41,14 @@ object FootballApi {
     @Singleton
     fun provideApiService(retrofit: Retrofit): FootballApiService =
         retrofit.create(FootballApiService::class.java)
+
+    @Provides
+    @Singleton
+    fun provideDatabase(app: Application) =
```

Comment:
application context could be provided this way `@ApplicationContext appContext: Context`